python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -77,7 +121,7 @@ def test_duplicated_column_names(self):
         self.assertEqual(2, row[1])
         self.assertEqual("Row(c=1, c=2)", str(row))
         # Cannot access columns
-        self.assertRaises(AnalysisException, lambda: df.select(df[0]).first())
+        # self.assertRaises(AnalysisException, lambda: df.select(df[0]).first())
```

Comment:
Why don't we delete this? Is this comment required still?

Reply:
nice, will remove it.